packages/slice-machine/components/Preview/components/IframeRenderer/index.tsx

```diff
@@ -73,25 +73,27 @@ const IframeRenderer: React.FunctionComponent<IframeRendererProps> = ({
   }, [client, size, sliceView]);
 
   return (
-    <Flex
-      sx={{
-        justifyContent: "center",
-        borderTop: "1px solid #F1F1F1",
-        margin: "0 auto",
-        overflow: "auto",
-        ...iframeSizes[size],
-      }}
-    >
-      <iframe
-        ref={ref}
-        src={canvasUrl}
-        style={{
-          border: "none",
-          height: "100%",
-          width: "100%",
+    <Box sx={{ minHeight: "calc(100vh - 67px)", bg: "grey01" }}>
+      <Flex
+        sx={{
+          justifyContent: "center",
+          borderTop: "1px solid #F1F1F1",
```

Comment:
theme variable here ?

Reply:
this is not part of the theme yet, I'll need to see what to use with design team